.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Report a bug
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: rusmonster
+
+---
+
+** Please provide the application logs when reporting bugs **
+
+1. Enable debug log
+
+    ```
+    // Before creating a new Conversations Client with create() add this line:
+    ConversationsClient.setLogLevel(android.util.Log.VERBOSE);
+    ...
+
+    ConversationsClient.create(.... your usual init code
+    ```
+
+2. Run your application and reproduce the problem
+
+3. Capture the entire device log by running
+
+    ```
+    adb logcat -d > my_error_log
+    ```
+
+4. Send my_error_log together with this bug report.
```

Comment:
Won't this have any sensitive information?

Reply:
It most probably will reveal some data. Sending this through support site is safer, but more cumbersome.

Lets add a note that if the customer is concerned with the privacy they could send via other channels?